spyder_kernels/console/start.py

```diff
@@ -293,6 +348,17 @@ def main():
     while '' in sys.path:
         sys.path.remove('')
 
+    # Handle conda environment. See Spyder issue #9077
```

Comment:
Let's change this comment to

    # Add to PATH most of the directories added when activating a conda
    # environment.
    # See Spyder issue #9077.

Reply:
@ccordoba12 With URL awareness work soon in place, it would really help when browsing the code. How about adding the full link?

```python
# Add to PATH most of the directories added when activating a conda
# environment.
# See https://github.com/spyder-ide/spyder/issues/9077
```